main.py

```diff
@@ -56,26 +61,26 @@ def create_character():
         option = input("Enter the number of your choice: ")
         
         if option == '1':
-            try:
-                amount = int(input("Enter the amount to increase: "))
-                character.increase_stat(amount)
-            except ValueError:
-                print("Please enter a valid number.")
+            while True:
+                try:
+                    amount = int(input("Enter the amount to increase: "))
+                    character.increase_stat(amount)
+                    break  # Exit the stat increase loop if successful
+                except ValueError:
+                    print("Please enter a valid number.")
         
         elif option == '2':
             character.level_up()
         
         elif option == '3':
-            print("\nCurrent Stats:")
-            print(character.display_stats())
+            display_stats(character)
         
         elif option == '4':
-            print("Exiting...")
+            print("Thank you for playing! Exiting the game...")
             break
         
         else:
             print("Invalid option. Please choose again.")
 
 if __name__ == "__main__":
```

Comment:
I dont think this is necessary. Just main() should work if it doesn't then ignore this comment. Should add a comment about if statement